### DISCUSSION TITLE
Custom DNS Endpoints

### DISCUSSION THREAD

**Discussion by superteece** [BishopFox/sliver#733] [Q&A] [answered]
Is there a way to use nameservers/subdomains other than _ns1_. and _1_.?

**Comment by superteece** [comment, marked as answer]
I re-read the wiki for a third or fourth time and saw the "arbitrary" part about the sub-domain this time.

My question remains open for the nameserver though. Can something other than _ns1_ be used?

*ANSWER UPDATE WITH SOLUTION*
Here's the bind9 config that is working at this moment:
```
@    IN    A    "Sliver IP"
ns1  IN    A    "Bind9 IP"
...
ns3  IN    A    "Sliver IP"
1     IN    NS   ns3.example.com.
```

**Reply by szymex73** [reply]
NS1 is just a name of an A record that will be used for the NS record, you should be able to change it to whatever you want as long as you reflect it in the respective NS record too

**Reply by moloch--** [reply]
This is correct you can name the records anything you want so long as they're valid they'll work.

**Reply by superteece** [reply]
So then can I leave _ns1_ pointing to my actual domain namesever's IP, and make a _ns3_ A record that points to my Sliver server?

The resulting config on my Bind9 server would be:

```
@    IN    A    "Sliver IP"
ns1  IN    A    "Bind9 IP"
...
ns3  IN    A    "Sliver IP"
1     IN    NS   "Sliver IP"
```

**Reply by moloch--** [reply]
Ahh yes, I think that works though I'm not sure of the exact Bind config.

**Reply by superteece** [reply]
Here's the bind9 config that is working at this moment:

```
@    IN    A    "Sliver IP"
ns1  IN    A    "Bind9 IP"
...
ns3  IN    A    "Sliver IP"
1     IN    NS   ns3.example.com.
```
